src/xml/xml.cpp

```diff
@@ -857,9 +859,23 @@ bool wxXmlDocument::Load(wxInputStream& stream, int flags)
             wxLogError(_("XML parsing error: '%s' at line %d"),
                        error.c_str(),
                        (int)XML_GetCurrentLineNumber(parser));
+            if (err_details)
```

Comment:
I would still put `wxLogError()` in `else` part of this check for the reasons previously discussed.

Maybe I'm missing something, could you please explain why do you think it would be useful to both log and return the error?

Reply:
No reason I can think of other then my tendency to avoid touching existing code without a good reason. Presumably anyone that uses the structure will have their own message without needing the log message, or they could just call wxLogError themselves.